upstream.Dockerfile

```diff
@@ -17,8 +17,8 @@ COPY go.sum go.sum
 COPY cmd cmd
 COPY util util
 COPY test test
-RUN make build
-RUN make build-util
+RUN CGO_ENABLED=0 make build
```

Comment:
Is there a reason we need to disable CGO here? Does an updated dependency require it?

Additionally, the downstream `Dockerfile` explicitly enables CGO.

Reply:
The upstream dockerfile image will not build otherwise:
docker build -t quay.io/agreene/olm:webhook-new -f upstream.Dockerfile .
Sending build context to Docker daemon  100.9MB
Step 1/28 : FROM golang:1.13-alpine as builder
 ---> 0e5426771795
Step 2/28 : LABEL stage=builder
 ---> Using cache
 ---> 3531687a249c
Step 3/28 : WORKDIR /build
 ---> Using cache
 ---> b7b246aa5d08
Step 4/28 : RUN apk update && apk add bash make git mercurial jq && apk upgrade
 ---> Using cache
 ---> 0d62b0495a72
Step 5/28 : COPY .git/HEAD .git/HEAD
 ---> Using cache
 ---> ab84e548599c
Step 6/28 : COPY .git/refs/heads/. .git/refs/heads
 ---> 5696581301a4
Step 7/28 : RUN mkdir -p .git/objects
 ---> Running in 9546e89f28c6
Removing intermediate container 9546e89f28c6
 ---> 55607d071b85
Step 8/28 : COPY Makefile Makefile
 ---> d33c08a8c47f
Step 9/28 : COPY OLM_VERSION OLM_VERSION
 ---> a934ee7e6ba2
Step 10/28 : COPY pkg pkg
 ---> 94b52e4fc3ef
Step 11/28 : COPY vendor vendor
 ---> 7e489b00274a
Step 12/28 : COPY go.mod go.mod
 ---> 52a0b3207e73
Step 13/28 : COPY go.sum go.sum
 ---> a5c109d58075
Step 14/28 : COPY cmd cmd
 ---> 33bdad10f285
Step 15/28 : COPY util util
 ---> 118c343f87c3
Step 16/28 : COPY test test
 ---> 7bda269d5a01
Step 17/28 : RUN make build
 ---> Running in d336e22af4f6
go build -mod=vendor -ldflags "-X github.com/operator-framework/operator-lifecycle-manager/pkg/version.GitCommit=4e093603eb621856a26ed6c507aec54733de93d4 -X github.com/operator-framework/operator-lifecycle-manager/pkg/version.OLMVersion=`cat OLM_VERSION`" -o bin/catalog github.com/operator-framework/operator-lifecycle-manager/cmd/catalog
# github.com/opencontainers/runc/libcontainer/system
exec: "gcc": executable file not found in $PATH
make: *** [Makefile:104: github.com/operator-framework/operator-lifecycle-manager/cmd/catalog] Error 2
The command '/bin/sh -c make build' returned a non-zero code: 2
```